internal/plugin.go

```diff
@@ -587,6 +587,76 @@ func (p *Plugin) assertValidConfig() error {
 		}
 	}
 
+	for i := range p.PolicySets {
+		seen := map[string]bool{}
+		plcset := &p.PolicySets[i]
+
+		if plcset.Name == "" {
+			return fmt.Errorf(
+				"each policySet must have a name set, but did not find a name at policySet array index %d", i,
+			)
+		}
+
+		if seen[plcset.Name] {
```

Comment:
@ycao56 could you please declare a new set for `seen` in the event the policy set and policy name are the same?

Reply:
sure!

Reply:
`seen` is reassigned on line 591. So it should be safe?

Reply:
As discussed in chat, this needs to be defined outside of the loop.